### ISSUE TITLE
Add support for --set on cli

### ISSUE THREAD

**Issue by cyberox** [jkroepke/helm-secrets#266]
### Problem Statement

I would like to be able to provide the vals backend path to the helm chart as `--set` command eg:

`helm secrets template bitnami/mysql --name-template mysql --set auth.rootPassword=ref+vault://secret/mysql#/rootPassword`

Currently the value is not replaced with the value from vault.

### Proposed Solution

support `--set` option

### Environment

- Helm Version: 3.9.4
- Helm Secrets Version: 4.0.0
- OS: Mac
- Shell: zsh

**Comment by jkroepke** [comment]
Thats an interesting idea ...

**Comment by jkroepke** [comment]
Feature implemented in #266

In additional to 

`--set auth.rootPassword=ref+vault://secret/mysql#/rootPassword`

I have also implement 

`--set-file auth.rootPassword=secrets+literal://ref+vault://secret/mysql#/rootPassword` 

for wrapper-less environments like ArgoCD.

Do you have a chance to test this?

```
helm plugin install https://github.com/jkroepke/helm-secrets --version vals-literal
```

I would also much appreciate it, if its enough documented form user perspective.

**Comment by cyberox** [comment]
That was fast!
I tested the --set command and it is working fine, I also added it to our argocd instance, and using the secrets+literal I can finally fetch secrets from vault using helm chart paramters.
Great job, and thank you for the feature!
